examples/use-cases/src/encrypt.ts

```diff
@@ -0,0 +1,43 @@
+import { VaultClient } from "@piiano/vault-client";
+
+async function main() {
+  const client = new VaultClient({
+    vaultURL: "http://localhost:8123",
+    apiKey: "pvaultauth",
+  });
+
+  // a collection is needed for validation and scoping purposes even though it doesn't store data
+  await client.collections.addCollection({
+    requestBody: {
+      name: "users",
+      type: "PERSONS",
+      properties: [
+        { name: "name", data_type_name: "NAME" },
+        { name: "email", data_type_name: "EMAIL" },
+        { name: "phone", data_type_name: "PHONE_NUMBER" },
+      ],
+    },
+  });
+
+  var ciphertext = await client.crypto.encrypt({
+    collection: "users",
+    reason: "AppFunctionality",
+    requestBody: [
+      {
+        type: "deterministic",
```

Comment:
`type` has a default of `randomized` so if you don't care about the example showing specifically deterministic encryption we can just omit `type`.
```suggestion
```

Reply:
removing randomized as it is the default